src/UglyToad.PdfPig/Geometry/PdfPath.cs

```diff
@@ -638,6 +714,38 @@ internal void Rectangle(decimal x, decimal y, decimal width, decimal height)
             LineTo(x + width, y + height);
             LineTo(x, y + height);
             LineTo(x, y);
+            IsDrawnAsRectangle = true;
+        }
+
+        /// <summary>
+        /// Order matters
```

Comment:
I'd add a bit of extra explanation here, e.g. `Compares 2 PdfPaths for equality. Paths will only be considered equal if the commands which construct the paths are in the same order`. And also an `<inheritdoc/>`.

Reply:
Agreed. just not sure what you mean regarding the `<inheritdoc/>` :D